.github/workflows/performance.yml

```
@@ -70,16 +98,30 @@ jobs:
           docker-compose run --rm php php -i
           docker-compose run --rm php locale -a
 
-      - name: Install MU plugin
-        run: |
-          mkdir ./src/wp-content/mu-plugins
-          cp ./tests/performance/mu-plugins/server-timing.php ./src/wp-content/mu-plugins/server-timing.php
-
       - name: Install WordPress
         run: npm run env:install
 
+      - name: Install wordpress importer plugin and import dummy data
+        run: |
+          npm run env:cli -- plugin install wordpress-importer --activate
+          curl -O https://raw.githubusercontent.com/WPTT/theme-unit-test/master/themeunittestdata.wordpress.xml
```

Let's use the XML file from a specific hash to ensure that changes to the files don't affect our results.
```suggestion
          curl -O https://raw.githubusercontent.com/WPTT/theme-test-data/b9752e0533a5acbb876951a8cbb5bcc69a56474c/themeunittestdata.wordpress.xml
```